lib/jobs/secure-erase-drive.js

```diff
@@ -407,7 +407,7 @@ function secureEraseJobFactory(
         }, []);
 
         // add script's url in the first command
-        this.commands[0].downloadUrl = '/api/1.1/templates/secure_erase.py';
+        this.commands[0].downloadUrl = '/api/current/templates/secure_erase.py';
```

Comment:
@benbp : {{ api.templates }} will duplicate the base url  and it won't work in this case

api.templates = self.renderContext.api.server + '/api/current/templates